scania-digital-design-system/sdds
# feat(native components): remove native components

**Describe pull-request**  

- Moves all native component css to dir tegel/native-components-DEPRECATED
- Updated tsconfig
- Corrected the name of the spinner story name
- Updated data-table and dropdown to use sdds-checkbox instead of native component.

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1470

**How to test**  
1. Go to storybook link below
2. Check that all the web components are still working.

**Suggested test steps**
